sdk/src/wallet/operations/balance.rs

```diff
@@ -58,6 +49,22 @@ where
             }
         }
 
+        // Add block issuance credits
+        for account in wallet_data.accounts() {
+            let account_id = *account.output.as_account().account_id();
+            // TODO: Maybe cache this in the wallet?
+            *balance.mana.block_issuance_credits.entry(account_id).or_default() += self
+                .client()
+                .get_account_congestion(&account_id)
+                .await?
+                .block_issuance_credits;
+        }
```

Comment:
The requests should be sent in parallel so it doesn't really take longer with multiple accounts

Reply:
tbh I think we shouldn't call this here anyway, since we're holding the wallet data hostage during this entire process.

Reply:
Why not move it to the beginning before then?

Reply:
Not sure about caching since the BIC also gets decayed all the time